### DISCUSSION TITLE
Visualize planned path, modify path with checkpoints

### DISCUSSION THREAD

**Discussion by RaptorSwire** [voxeltycoon/issues#507] [General]
Often I want to see which path is the train going to take, so I can avoid crowding.

Some visual overlay or showing path of selected vehicle (checkbox on selected vehicle) would help a lot!

In addition to this topic, sometimes I want the train to avoid using certain path, even though it is the shortest, to avoid crowding. Setting checkpoint (in TTD called Waypoint) the train/car HAS TO take on the way would be great. This feature already exists when you use train station and set train to just pass through it, but it is not visually very pleasing, since it is not a station and takes unnecessary amount of space

**Comment by katepolak** [comment]
Partially related to #41
